### ISSUE TITLE
Tags, functions, objects...

### ISSUE THREAD

**Issue by adamcameron** [cybersonic/luceedocs#23]
There's some documentation for the items listed above, but what about the rest of the language like `component` (not `<cfcomponent>`, but just `component`), for(), try / catch etc? Those're pretty essential of a CFML docs site.

**Comment by DominicWatson** [comment]
Yes, I think eventually the docs should absolutely contain more than just that reference material. There should be a language specification as the next priority, followed by guides on various themes. All cross referenced in the reference docs.
